cmd/skywire-updater/commands/update.go

```diff
@@ -0,0 +1,69 @@
+package commands
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+
+	"github.com/skycoin/skywire-updater/internal/pathutil"
+	"github.com/skycoin/skywire-updater/pkg/api"
+	"github.com/skycoin/skywire-updater/pkg/store"
+	"github.com/skycoin/skywire-updater/pkg/update"
+	"github.com/spf13/cobra"
+)
+
+const configEnv = "SW_UPDATER_CONFIG"
+
+var defaultConfigPaths = [3]string{
```

Comment:
I would prefer:
```golang
var defaultPaths = pathutil.UpdaterDefaults()

/* ... */
// 3. %s`, defaultPaths[pathutil.WorkingDirLoc], defaultPaths[pathutil.HomeLoc], defaultPaths[pathutil.LocalLoc]),

```

Reply:
Thank you for bringing this up. Resolved.